<?xml version="1.0"?>
<launch>




  <group ns="global">
      <param name="robot_name" value="global" />
      <param name="global_map" value="global" />

      <node pkg="setup" type="map_percent.py" name="map" output="screen" />
  </group>



</launch>
